alicloud/table_alicloud_vpc_route_entry.go

```diff
@@ -0,0 +1,253 @@
+package alicloud
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/vpc"
+)
+
+//// TABLE DEFINITION
+
+func tableAlicloudVpcRouteEntry(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "alicloud_vpc_route_entry",
+		Description: "Alicloud VPC Route Entry",
+		List: &plugin.ListConfig{
+			ParentHydrate: listVpcRouteTable,
+			Hydrate:       listVpcRouteEntries,
+		},
+		// Get: &plugin.GetConfig{
```

Comment:
Why do we have this commented out code here?

Reply:
removed as discussed